README.md

```diff
@@ -37,28 +42,28 @@ This will be the file that holds your S3 and SQS configurations.
       }
     }
     ```
-1. [Register or create a new GitHub organization](https://github.com/settings/organizations). Find your organization's ID by visiting `https://api.github.com/orgs/<your-org-name>` and copying the `id` into the whitelist of `organizations` in `config/local.js`.
+1. [Register or create a new GitHub organization](https://github.com/settings/organizations) with a name of your choosing. Then find your organization's ID by visiting `https://api.github.com/orgs/<your-org-name>` and copying the `id` into the whitelist of `organizations` in `config/local.js`.
     ```js
     organizations: [
       99999999 // your org added here
     ]
     ```
     The organization will need to grant access to federalist, which can be done during:
       * a first-time login with your GitHub credentials
-      * in the Settings -> Applications -> federalist view in your GitHub Account
-1. Type `cf login --sso -a https://api.fr.cloud.gov -o gsa-18f-federalist -s staging` in terminal.
-1. Visit https://login.fr.cloud.gov/passcode to get a one time passcode.
-1. Enter your passcode back into terminal.
+      * in the [Authorized OAuth Apps](https://github.com/settings/applications) tab in your GitHub Account settings
+1. Paste `cf login --sso -a https://api.fr.cloud.gov -o gsa-18f-federalist -s staging` into your terminal window.
+1. Visit https://login.fr.cloud.gov/passcode to get a Temporary Authentication Code.
+1. Paste this code into the terminal, and hit the return key. (For security purposes, the code won't be rendered in the terminal.)
 1. Type `cf apps`.
 1. Type `cf env federalist-staging` to get environment variables.
-1. Find the section in the listing of environment variables that starts with `"s3": [` and look for the following values and paste those values into the S3 section in your `local.js` file.
-    - `access_key_id`
-    - `bucket`
-    - `secret_access_key`
-1. Find the section in the listing of environment variables for SQS and look for the following values and paste those values into the SQS section in your `local.js` file.
+1. Open your `local.js` file and find the `sqs` environment variables section. Then paste the following values from the terminal:
     - `FEDERALIST_AWS_BUILD_KEY` is `accessKeyId`
     - `FEDERALIST_SESSION_SECRET` is `secretAccessKey`
     - `FEDERALIST_SQS_QUEUE` is `queue`
+1. Next find the `s3` section of the file and paste these values from the terminal (leave the `region` value as `'us-gov-west-1'`):
```

Comment:
I liked the guidance to find the section in your terminal that starts with `"s3": [` 
Those keys/values are different for `s3` than they are for `sqs` (those are two different Amazon services that Federalist uses, and they have different sets of credentials).

Reply:
I flopped the order of these two steps because the `sqs` section of the `local.js` file comes before the `s3` section. Could you take a crack at editing this part for me? :)